### PR TITLE
Landing page more mobile frieldly update

### PR DESCRIPTION
**Pull request by mjadach-iv**
Now: 
image

Before:
image

### CODE REVIEW

File: src/future-hopr-lib-components/Layout/index.tsx [hunk 1]
```diff
@@ -70,6 +70,7 @@ const Layout: React.FC<{
   };
   drawerType?: 'blue' | 'white';
   drawerItems: ApplicationMapType;
+  drawerFunctionItems? : any;
```
Comment (diegoalzate) [inline src/future-hopr-lib-components/Layout/index.tsx:73]:
```suggestion
  drawerFunctionItems? : ApplicationMapType;
```

Comment (diegoalzate) [reply src/future-hopr-lib-components/Layout/index.tsx:73]: I think this is the type you use for this throughout the app right ?

**Review by diegoalzate — CHANGES_REQUESTED** [review]
Small change and then looks good to merge

File: src/future-hopr-lib-components/Layout/index.tsx (hunk 1 above)
Comment (mjadach-iv) [reply src/future-hopr-lib-components/Layout/index.tsx]: Fixed

**Review by diegoalzate — APPROVED** [review]

**Pull request merged**